client/src/templates/Challenges/classic/mobile-layout.tsx

```diff
@@ -61,7 +67,9 @@ class MobileLayout extends Component<MobileLayoutProps> {
 
     // Unlike the desktop layout the mobile version does not have an ActionRow,
     // but still needs a way to switch between the different tabs.
-    const displayEditorTabs = showUpcomingChanges && usesMultifileEditor;
+    const projectBasedChallenge = showUpcomingChanges && usesMultifileEditor;
+
+    const eventKeys = [1, 2, 3, 4, 5];
```

Comment:
The reason for this was that the `activeKey` and `defaultActiveKey` for the `Tabs` are `1`. The first `TabPane` is the instruction pane, which isn't always shown. So this sets the `eventKey` to `1` for the first `TabPane` that's shown. But I played with it a bit - I don't any difference between using a string, number, or duplicate values as the `eventKey`. No errors or nothin. So it seems like it really doesn't matter.

Reply:
Well, I didn't do anything with this one for now. If you have more thoughts, I can change it up - or we can get it in like this and visit it later. It's probably not ideal.

Reply:
As I understand it, they're just react keys by another name - the only requirement being uniqueness amongst siblings.

Reply:
So, yeah, I'd rather we just used some keys rather than the shift stuff.  It's minor, but doing it that way makes it feel like it has to be dynamic (and it doesn't).

Reply:
On second thoughts, do you mind if we just get rid of these?  I think they're okay as is, but dynamically generated keys could easily lead to bugs later.

Reply:
I've got another PR ready to go to fix this after we get this merged